core/zms/src/main/rdl/Authority.rdli

```diff
@@ -0,0 +1,19 @@
+// Copyright 2021 Athenz Authors
+// Licensed under the terms of the Apache version 2.0 license. See LICENSE file for terms.
+
+//List of valid user authority attribute values
+type UserAuthorityValidValuesList Struct {
```

Comment:
let's replace the Valid since that's kind of implied. So maybe we can use:

UserAuthorityAttributeValueList